include/swift/AST/SubstitutionMap.h

```diff
@@ -63,10 +63,16 @@ class SubstitutionMap {
 
   void addSubstitution(CanType type, Type replacement);
 
+  const llvm::DenseMap<TypeBase *, SmallVector<ParentType, 1>> &getParentMap() const {
```

Comment:
This isn't actually used anywhere that I can tell. Can we remove it? The `dump()` method can just poke at `parentMap` directly.